public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+/*---general page and components css---*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+* {
+  overflow: auto;
+}
+
+body {
+  width: 100%;
+  margin: auto;
+  font-family: Helvetica, Arial, "Lucida Grande", sans-serif;
+  font-weight: 300;
+}
+
+nav {
+  margin-left: -1em;
+}
+nav li {
+  display: inline-block;
+}
+
+nav, section {
+  text-align: center;
+ }
+header > p {
+    margin-bottom: -.7em;
+  }
+section {
+  width: 100%;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 2em;
+}
+
+ul {
+    margin-left: -1em;
```

Comment:
Again with negative margins. Also, be consistent with your whitespace; here you have 4 space indent, elsewhere you have 2.